tests/user/users.json

```diff
@@ -0,0 +1,3504 @@
+{
```

Comment:
This is created within the test. What about having only "{}" as content?

Reply:
But the pre-generated file will stop `create_users_json.yml` from creating the proper file with valid expiration dates.
Should the file still be missing instead?

Reply:
This is a mistake on my side. This file should not make into the patch.

Reply:
Fixed now. I removed the file.